Automattic/jetpack
# Security Settings: Convert to feature checks

#### Changes proposed in this Pull Request:

* Introduces `hasActiveSiteFeature` selector.
* Replaces plan check in Security settings with feature checks.

#### Jetpack product discussion

Internal: p9dueE-4T6-p2

#### Does this pull request change what data or activity we track or use?


No

#### Testing instructions:






* Go to `/wp-admin/admin.php?page=jetpack#/settings` on your test site
* Make sure the "Backups and security scanning" section shows the accurate date, based on plan level.
